src/main/java/io/strimzi/kafka/quotas/VolumeDetails.java

```diff
@@ -0,0 +1,33 @@
+/*
+ * Copyright 2021, Red Hat Inc.
```

Comment:
I don't think you should be including any Red Hat copyright headers in new files. You should use something like this:

```
/*
 * Copyright Strimzi authors.
 * License: Apache License 2.0 (see the file LICENSE or http://apache.org/licenses/LICENSE-2.0.html).
 */
```

Reply:
Thanks, I did wonder about that but was copying the existing files. 

I'll update my changes to use strimizi authors.